tests/fuzz/fuzz.go

```diff
@@ -0,0 +1,38 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build gofuzz
+
+package fuzz
+
+import (
+	"github.com/istio/istio/pilot/pkg/config/kube/crd"
```

Comment:
should be `istio.io/istio`

Reply:
In the current build `istio.io/istio` results in "cannot find package..." for both imports.

Reply:
With this code running `go-fuzz-build` fails complaining about the import for me.

Reply:
Aha. I am running this through OSS-fuzz and there might be a bit of difference there. Do you prefer the fuzzers to be compile'able with go-fuzz-build before merging? It could also be a solution to make a rule in the `Makefile` that compiles the fuzzers

Reply:
It would be useful to have some way to run it locally, whether its through `go-fuzz-build` or otherwise, since it will be useful for debugging/dev

Reply:
Sure, no problem. Did you get istio with `go get github.com/istio/istio/...` like in the Dockerfile?

Reply:
No, I have it cloned into `go/src/istio.io/istio`. I don't think it will build if you have it under github due to how go building works?

Reply:
I see the issue locally. I will work on a way to have it run locally with `go-fuzz-build` as well.

Reply:
I have updated the import paths.